helm/default-apps-vsphere/values.yaml

```diff
@@ -181,6 +181,18 @@ apps:
     # used by renovate
     # repo: giantswarm/observability-bundle
     version: 1.5.2
+  observabilityPolicies:
+    appName: observability-policies
+    chartName: observability-policies
+    catalog: default
+    clusterValues:
+      configMap: false
+      secret: false
+    dependsOn: kyverno-crds
+    namespace: kube-system
+    # used by renovate
+    # repo: giantswarm/observability-policies
```

Comment:
```suggestion
    # repo: giantswarm/observability-policies-app
```

Reply:
not sure this really matters, but seems like the repo is called "observability-policies-app"?